### ISSUE TITLE
A few suggestions for API changes before 1.0

### ISSUE THREAD

**Issue by samuell** [scipipe/scipipe#65]
```go
// Connections
proc2.In("in").Connect(proc1.Out("out"))
// ... should definitely be (reads very naturally ... almost like English):
proc2.In("in").From(proc1.Out("out"))
// ... and *possibly* (but not as readable, so will probably not do this):
proc2.In("in", proc1.Out("out"))

// Parameter ports
ParamInPort()
ParamOutPort()
// ... should be:
InParamPort()
OutParamPort()
// ... and in Process:
InParam()
OutParam()

Keys()
// should be
Tags()
// (But do in new branch, so as not to break existing workflows)
```

**Comment by samuell** [comment]
This and more is now implemented in develop, with:
- d92065a289964177b2e40994fd590ea93cdb0390
- 75512dd20c5e39217f4ecc390a170a4c1df15e80
- 0424377d607f43b00fd9bbc9cec56703ba884fcb
- 9dc83104a3924a6030b0a81967da656966b7fcee

**Comment by samuell** [comment]
Implemented in [SciPipe 0.7](https://github.com/scipipe/scipipe/releases/tag/v0.7) so closing.
